components/styles.js

```diff
@@ -49,6 +49,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: 5,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Im not sure about, the week row just flex based on the content inside of it. So if someone decides to change the font size for each each day they won't run into any issues with overlapping content.